main/src/cgeo/geocaching/CacheDetailActivity.java

```diff
@@ -1553,13 +1553,18 @@ private void uploadPersonalNote() {
                 public void call() {
                     final IConnector con = ConnectorFactory.getConnector(cache);
                     if (con.supportsPersonalNote()) {
-                        con.uploadPersonalNote(cache);
+                        final boolean success = con.uploadPersonalNote(cache);
+
+                        final Message msg = Message.obtain();
+                        final Bundle bundle = new Bundle();
+                        if (success) {
+                            bundle.putString(SimpleCancellableHandler.MESSAGE_TEXT, CgeoApplication.getInstance().getString(R.string.cache_personal_note_upload_done));
+                        } else {
+                            bundle.putString(SimpleCancellableHandler.MESSAGE_TEXT, CgeoApplication.getInstance().getString(R.string.cache_personal_note_upload_error));
+                        }
+                        msg.setData(bundle);
```

Comment:
I would have simply written:

```java
  bundle.putString(SimpleCancellableHandler.MESSAGE_TEXT, CgeoApplication().getInstance().getString(success ?
    R.string.cache_personal_note_upload_done : R.string.cache_personal_note_upload_error));
```

to avoid repeating the same code.